proposals/20200925-admins.md

```diff
@@ -0,0 +1,30 @@
+# Falco Infra Admins
+
+This document defines the roles and responsibilities of the Falco Infra Admins.
+
+## Terms & Definitions
+
+- 'the systems' -  an umbrella term refering to the collection of infrastructure and software as a service tools used by the Falco maintainers to develop, build, test, release, and distribute Falco.
+- Falco maintainers - the group of people listed in the OWNERS file in falcosecurity/falco
+- CNCF - The Cloud Native Computing Foundation
+- LF - The Linux Foundation
+
+## Motivation
+
+As part of modern open development, the Falco project is dependent on a considerable amount of development infrastructure. CI systems, code hosting systems, community websites, self-run infrastructure, and cloud services are all examples. Whenever possible, the use of role based access control is used to give the correct people access to the correct services, however in many cases a service account is used with a shared username and password. The group needing access and the number of systems to access are both growing and putting some policies in place will help the community scale effectively.
+
+## Goals
+- Define who is an "infra admin"
+- Define the responsibilities of an "infra admin"
+
+
+## Proposal
+
+- Infra admins are the individuals listed in the OWNERS file of falcosecurity/falco.
```

Comment:
```suggestion
- Infra admins are the individuals listed in the OWNERS file of falcosecurity/test-infra.
```

The OWNERS file in Falco repository declares the Falco maintainers.

The OWNERS file in this repo (test-infra) declares the Falco Infra maintainers.

Clearly, the two sets could also be disjoint. Thus, I would not use that OWNERS file to declare Falco Infra maintainers too.

Reply:
Changed